app/client/components/FormRenderer.ts

```diff
@@ -860,6 +858,11 @@ class RefRenderer extends BaseFieldRenderer {
             },
             matchTriggerElemWidth: true,
           }),
+          css.resetSelectButton(
+            icon('CrossSmall'),
```

Comment:
here we could add some alt text for screen reader users through an `aria-label` attribute. I'd say it would be great to repeat the field name in that label, like `Clear {label} value`.